lib/errdiff/errdiff.go

```diff
@@ -0,0 +1,34 @@
+package errdiff
```

Comment:
So if you ever want to make a library that's importable by tests only, you can give it a no release annotation at the top.
```
//+build !release
```

You can technically import it in a library, but it would fail on building.

Reply:
Since we use bazel, I've marked it as `testonly` there; this also fails the build if the library makes it into a non-testonly target. Is it necessary to do both?